src/App.tsx

```diff
@@ -14,210 +31,74 @@ export const App: React.FC = () => {
     }
   }, []);
 
+  useEffect(() => {
+    if (user) {
+      try {
+        getTodos(user.id)
+          .then(setTodos);
+      } catch {
+        setError('Can\'t load todo');
+      }
+    }
+  });
+
+  const filteredTodos = todos.filter(todo => {
+    switch (status) {
+      case 'All':
+        return todo;
+
+      case 'Active':
+        return !todo.completed;
+
+      case 'Completed':
+        return todo.completed;
+
+      default:
+        throw new Error('Invalid filter type');
+    }
+  });
+
+  const amountOfActive = todos.filter(
+    todo => !todo.completed,
+  ).length;
+
+  const amountofCompleted = filteredTodos.some(
+    todo => todo.completed,
+  );
+
+  const allCompleted = todos.filter(
+    todo => todo.completed,
+  ).length === filteredTodos.length;
```

Comment:
figure out how to improve code readable, u could create function helper or use variable